POEApi.Model/ProxyMapper.cs

```diff
@@ -209,6 +209,18 @@ internal class ProxyMapper
             {"Uul-Netol", BreachType.UulNetol}
         };
 
+        private static readonly Dictionary<string, LegionType> _legionMap = new Dictionary<string, LegionType>
+        {
+            //Splinters are called Timeless Eternal Empire Splinter
```

Comment:
We should make these comments more explicit.  Something like: ```Eternal Empire splinters are called "Timeless Eternal Empire Splinter".``` Similar change for the emblems comment (the comment only applies to the Eternal Empire emblem).

Reply:
I think it's clear enough.